views/shared/_flash_messages.haml

```diff
@@ -0,0 +1,4 @@
+-unless flash[:error].nil?
+  %div
+    .alert.alert-danger{role: "alert"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace